### PR TITLE
Show python and django version in env admin page.

### PR DESCRIPTION
**Pull request by rlr**
This changes the Environment admin page to show python and django version in addition to the wsgi stuff.

r?

### CODE REVIEW

File: kitsune/kadmin/admin.py [hunk 1]
```diff
@@ -49,10 +48,14 @@ def celery_settings(request):
 
 
 def env(request):
-    """Admin view that displays the wsgi env."""
-    return http.HttpResponse(u'<pre>%s</pre>' % (jinja2.escape(request)))
-
-admin.site.register_view('env', view=env, name='WSGI Environment')
+    """Admin view that displays env info."""
+    return render_to_response('kadmin/env_view.html', {
+        'request': request,
```
Comment (willkg) [inline kitsune/kadmin/admin.py]: In fjord, the request gets added to the jinja environment for every template rendering. Is that not true in kitsune?

**Comment by willkg** [comment]
Hey! Ur stealing my codez! r+

File: kitsune/kadmin/admin.py (hunk 1 above)
Comment (rlr) [inline kitsune/kadmin/admin.py]: These are django templates though. And now I am confused as to why it works on fjord and not here.

Comment (willkg) [inline kitsune/kadmin/admin.py]: Oh, right. We do that in django-land, too:

https://github.com/mozilla/fjord/blob/master/fjord/settings/base.py#L548

**Pull request merged**